docs/apache-airflow/howto/set-up-database.rst

```diff
@@ -198,12 +198,6 @@ in the Postgres documentation to learn more.
 
    Details in the `SQLAlchemy Changelog <https://docs.sqlalchemy.org/en/14/changelog/changelog_14.html#change-3687655465c25a39b968b4f5f6e9170b>`_.
 
-We recommend using the ``psycopg2`` driver and specifying it in your SqlAlchemy connection string.
-
-.. code-block:: text
-
-   postgresql+psycopg2://<user>:<password>@<host>/<db>
```

Comment:
> postgres+psycopg2 is a bad scheme

postgres**ql**+psycopg2 `!=` postgres+psycopg2

Reply:
https://docs.sqlalchemy.org/en/14/dialects/postgresql.html#module-sqlalchemy.dialects.postgresql.psycopg2

Reply:
Yes. `postgres+psycopg2` used to be possible in sqlalchemy but it's been deprecated and removed, So our recommendation is correct.